### PR TITLE
Fix embedding model handling with Easy RAG

### PR DESCRIPTION
**Pull request by jmartisk**
Fixes https://github.com/quarkiverse/quarkus-langchain4j/issues/786

### CODE REVIEW

**Review by geoand — APPROVED** [review]

File: embedding-stores/chroma/deployment/src/main/java/io/quarkiverse/langchain4j/chroma/deployment/ChromaProcessor.java [hunk 1]
```diff
@@ -43,7 +43,7 @@ public void createBean(
                         ParameterizedType.create(EmbeddingStore.class, ClassType.create(TextSegment.class)))
                 .defaultBean()
                 .setRuntimeInit()
-                .defaultBean()
+                .unremovable()
```
Comment (geoand) [inline embedding-stores/chroma/deployment/src/main/java/io/quarkiverse/langchain4j/chroma/deployment/ChromaProcessor.java:46]: Is this really correct?

Comment (jmartisk) [reply embedding-stores/chroma/deployment/src/main/java/io/quarkiverse/langchain4j/chroma/deployment/ChromaProcessor.java:46]: Now that the Dev UI page doesn't have a injection point for it anymore, I think we need to make it unremovable if we want to make it work for dev mode purposes. One test that uses Chroma was failing without this change

Comment (geoand) [reply embedding-stores/chroma/deployment/src/main/java/io/quarkiverse/langchain4j/chroma/deployment/ChromaProcessor.java:46]: At the very least, I would assume that we need the same thing everywhere

Comment (jmartisk) [reply embedding-stores/chroma/deployment/src/main/java/io/quarkiverse/langchain4j/chroma/deployment/ChromaProcessor.java:46]: Yeah right, I'll add it to others.. I think it's useful to have the embedding store working for Dev UI purposes even if the app doesn't have any injection point for it.

Comment (geoand) [reply embedding-stores/chroma/deployment/src/main/java/io/quarkiverse/langchain4j/chroma/deployment/ChromaProcessor.java:46]: Agreed

**Review by edeandrea — APPROVED** [review]
LGTM! Thanks for the quick turnaround!

**Pull request merged**